### PR TITLE
Filepath should have _sample at the end if it's a sample

### PR DESCRIPTION
**Pull request by odscjames**
https://github.com/open-contracting/kingfisher-scrape/issues/52

### CODE REVIEW

File: kingfisher_scrapy/pipelines.py [hunk 1]
```diff
@@ -41,7 +41,10 @@ def file_path(self, request, response=None, info=None):
         elif not media_ext or ('json' in content_type and media_ext != '.json'):
             media_ext = '.json'
         # Put files in a directory named after the scraper they came from, and the scraper starttime
-        return '%s/%s/%s%s' % (info.spider.name, start_time_str, media_guid, media_ext)
+        if hasattr(info.spider, 'sample') and info.spider.sample == 'true':
+            return '%s_sample/%s/%s%s' % (info.spider.name, start_time_str, media_guid, media_ext)
+        else:
```
Comment (BibianaC) [inline kingfisher_scrapy/pipelines.py]: the `else:` is not needed. If the `if` is false then it will move the next line.

Comment (odscjames) [reply kingfisher_scrapy/pipelines.py]: Oh yes, true. Tho personally I think it makes it easier to read - I find it clearer when skimming it

Comment (odscjames) [reply kingfisher_scrapy/pipelines.py]: I know that's very subjective :-)

Comment (jpmckinney) [reply kingfisher_scrapy/pipelines.py]: FYI, in most of our (and others') Python code, `else` clauses aren't used in such cases.

**Pull request merged**